### DISCUSSION TITLE
[Suggestion] Credits for arrested hostiles.

### DISCUSSION THREAD

**Discussion by AlanGuardsman** [FakeFishGames/Barotrauma#9919] [Ideas and suggestions]
Simple as that - if you have terrorists and/or pirates KO'd non-lethally, then cuffed and delivered to the end of the mission you'll get additional reward like [Reward = kx; where k is reward and x is amount of arrestants, simply as that].

Additionally, if current game build and engine allow that, it is possible to make this feature a little bit harder and, if the mission end not at an outpost, you'll still have arrestans with you and to get rewards you'd actually need to reach an actual outpost to give them in, turning it into a prisoner escort additional mission (practically, not mechanically).

**Comment by gerakis** [comment]
There is a security talent that does exactly this, called By the Book. In my opinion the reward is not nearly worth the risk involved. The reward needs to be several times larger in my opinion. I dont think ive ever once seen anyone actually do it in multiplayer. The only time you ever really get a chance to use it is at abandoned outposts. 

You could also try to arrest pirates, but thats crazy-man tier though. Thats insanely dangerous. Maybe if you had some kind of stun-depth charge that knocked them out for a while.

**Reply by mygamingaccount** [reply]
Use 40mm stun grenades when you board the pirate sub
Or just heal two of them before they bleed out.
It's a fun challenge!

So why is this a suggestion anyway? It's in the game already.
